dev/ese/BUILDING.md

```diff
@@ -0,0 +1,76 @@
+> Shell commands in this document are given in [PowerShell](https://github.com/PowerShell/PowerShell)
```

Comment:
So this file is in dev\ese\BUILDING.md ... should it be at the root of the github repo tree, next to CONTRIBUTING.md?

Reply:
Sure, I'll move it to the root